assets/js/apps/metadata-settings/components/Modals/DeleteFieldModal.js

```diff
@@ -15,7 +15,7 @@
 
 import { Modal, Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { Icon, warning } from '@wordpress/icons';
+import { Icon, error } from '@wordpress/icons';
```

Comment:
Comment: Interesting they dropped support for `warning`. Using `error` I think is the closest replacement. How did you find this this issue? For me it seemed to build fine using `warning` but only gave an error when I actually opened the modal in a browser.